ci-operator/step-registry/cluster-profiles/cluster-profiles-config.yaml

```diff
@@ -463,6 +463,7 @@
         - cluster-api-provider-gcp
         - cluster-etcd-operator
         - installer
+        - kueue-operator
```

Comment:
As what we discussed on slack, the `gcp-qe` cluster profile is only for openshift-qe specific ci jobs, I think you can use other cluster profiles, e.g: `gcp-3`, `gcp-openshift-gce-devel-ci-2`, `gcp-observability`.

Reply:
@jianlinliu https://github.com/openshift/release/pull/67003 has already been included in this PR.

Reply:
It was caused by mirror-registry cert mismatch issue, now it should be fixed, and I verified it in https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/67044/rehearse-67044-periodic-ci-openshift-verification-tests-main-installer-rehearse-4.19-installer-rehearse-gcp/1945051818291630080, it work well. You can try again.